entity-framework/core/providers/sql-server/functions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Function Mappings - Microsoft SQL Server Database Provider - EF Core
+description: Function Mappings of the Microsoft SQL Server database provider
+author: bricelam
+ms.date: 10/07/2020
+uid: core/providers/sql-server/functions
+---
+# Function Mappings of the Microsoft SQL Server Provider
+
+This page shows which .NET members are translated into which SQL functions when using the SQL Server provider.
+
+## Binary functions
+
+.NET                         | SQL                           | Added in
+---------------------------- | ----------------------------- | --------
+bytes.Contains(value)        | CHARINDEX(@value, @bytes) > 0 | EF Core 5.0
```

Comment:
FYI at least for Npgsql I'll be omitting the parameter placeholder (so `value` instead of `@value`). This is because that's not the SQL that actually gets sent to PG (placeholders are positional: `$1`, `$2`), and it seems clear enough that `value` from the left side gets interpolated into the right side.

Reply:
I mostly put these in to differentiate between contextual keywords like `week` and variables.